CI/CD Stability

I've been seeing some stability issues with our CI/CD and this issue is to track the various errors to eventually patch:

- [ ] test_elem_disp_plot (should be resolved in #758)
- [ ] test_load_array
- [ ] examples/00-mapdl-examples... [ 45%] geometry.py

#### test_elem_disp_plot
```
tests/test_post.py::test_elem_disp_plot[False-Z] Fatal Python error: Segmentation fault

Thread 0x00007f92c78a6700 (most recent call first):
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/threading.py", line 306 in wait
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/threading.py", line 558 in wait
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/tqdm/_monitor.py", line 60 in run
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/threading.py", line 932 in _bootstrap_inner
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/threading.py", line 890 in _bootstrap

Current thread 0x00007f92dd9a8740 (most recent call first):
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/ansys/mapdl/reader/mesh.py", line 635 in fix_missing_midside
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/ansys/mapdl/reader/mesh.py", line 144 in _parse_vtk
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/ansys/mapdl/core/mesh_grpc.py", line 472 in _grid
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/ansys/mapdl/reader/mesh.py", line 67 in _surf
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/ansys/mapdl/core/post.py", line 949 in _plot_cell_scalars
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/ansys/mapdl/core/post.py", line 572 in plot_element_displacement
  File "/home/runner/work/pymapdl/pymapdl/tests/test_post.py", line 605 in test_elem_disp_plot
```


#### test_load_array
```
   def test_load_array(mapdl):
    
        dimx = np.random.randint(1, 15)
        dimy = np.random.randint(1, 15)
        my_conv = np.random.rand(dimx, dimy)
    
        mapdl.load_array("my_conv", my_conv)
>       assert np.allclose(mapdl.parameters["my_conv"], my_conv)
E       assert False
E        +  where False = <function allclose at 0x7f8637557e50>(array([0.46018965, 0.92870119, 0.97001612, 0.18044393, 0.37025709,\n       0.41013987, 0.3103413 , 0.85715418, 0.89454112, 0.5891909 ]), array([[0.46018965],\n       [0.92870119],\n       [0.97001612],\n       [0.18044393],\n       [0.37025709],\n       [0.41013987],\n       [0.3103413 ],\n       [0.85715418],\n       [0.89454112],\n       [0.5891909 ]]))
E        +    where <function allclose at 0x7f8637557e50> = np.allclose

tests/test_mapdl.py:780: AssertionError
```

#### test_plot_nodal_principal_total_strain
```
tests/test_post.py::test_plot_nodal_principal_total_strain[False] Fatal Python error: Segmentation fault

Thread 0x00007f2b29132700 (most recent call first):
  File "/usr/lib/python3.8/threading.py", line 306 in wait
  File "/usr/lib/python3.8/threading.py", line 558 in wait
  File "/home/runner/.local/lib/python3.8/site-packages/tqdm/_monitor.py", line 60 in run
  File "/usr/lib/python3.8/threading.py", line 932 in _bootstrap_inner
  File "/usr/lib/python3.8/threading.py", line 890 in _bootstrap

Current thread 0x00007f2b3ef8b740 (most recent call first):
  File "/home/runner/.local/lib/python3.8/site-packages/ansys/mapdl/reader/mesh.py", line 635 in fix_missing_midside
  File "/home/runner/.local/lib/python3.8/site-packages/ansys/mapdl/reader/mesh.py", line 144 in _parse_vtk
  File "/home/runner/.local/lib/python3.8/site-packages/ansys/mapdl/core/mesh_grpc.py", line 472 in _grid
  File "/home/runner/.local/lib/python3.8/site-packages/ansys/mapdl/reader/mesh.py", line 67 in _surf
  File "/home/runner/.local/lib/python3.8/site-packages/ansys/mapdl/core/post.py", line 908 in _plot_point_scalars
  File "/home/runner/.local/lib/python3.8/site-packages/ansys/mapdl/core/post.py", line 1869 in plot_nodal_total_principal_strain
  File "/home/runner/work/pymapdl/pymapdl/tests/test_post.py", line 370 in test_plot_nodal_principal_total_strain
```

#### Documentation build

```
...
generating gallery for examples/00-mapdl-examples... [ 45%] geometry.py
Segmentation fault (core dumped)
make: *** [Makefile:19: html] Error 139
make: Leaving directory '/home/runner/work/pymapdl/pymapdl/doc'
Error: Process completed with exit code 2.
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -28,15 +28,15 @@ jobs:
         with:
           python-version: 3.8
 
-      - name: Install requirements
-        run: pip install -r requirements_style.txt --disable-pip-version-check
-
-      - name: Spelling
-        run: make codespell
+      - name: Style
+        run: |
+          pip install -r requirements_style.txt --disable-pip-version-check
+          make
 
       - name: flake8
         if: always()
-        run: make flake8
+        run: |
+          make flake8
 
   testimport:
     name: Smoke Tests

--- a/ansys/mapdl/core/mapdl.py
+++ b/ansys/mapdl/core/mapdl.py
@@ -1494,7 +1494,7 @@ class _MapdlCore(Commands):
 
     def _get(self, *args, **kwargs):
         """Simply use the default get method"""
-        return float(self.get(*args, **kwargs))
+        return self.get(*args, **kwargs)
 
     def add_file_handler(self, filepath, append=False, level="DEBUG"):
         """Add a file handler to the mapdl log.  This allows you to

--- a/ansys/mapdl/core/mesh_grpc.py
+++ b/ansys/mapdl/core/mesh_grpc.py
@@ -173,7 +173,7 @@ class MeshGrpc(Mesh):
         >>> mapdl.mesh.n_node
         7217
         """
-        return int(self._mapdl.get_value(entity="NODE", item1="COUNT"))
+        return int(self._mapdl.get(entity="NODE", item1="COUNT"))
 
     @property
     @supress_logging
@@ -185,7 +185,7 @@ class MeshGrpc(Mesh):
         >>> mapdl.mesh.n_elem
         1520
         """
-        return int(self._mapdl.get_value(entity="ELEM", item1="COUNT"))
+        return int(self._mapdl.get(entity="ELEM", item1="COUNT"))
 
     @property
     def node_angles(self):
@@ -288,18 +288,18 @@ class MeshGrpc(Mesh):
         return self._node_coord
 
     def _load_nodes(self, chunk_size=DEFAULT_CHUNKSIZE):
-        """Loads nodes from server using the gRPC backend.
+        """Loads nodes from server.
 
         Parameters
         ----------
-        chunk_size : int, optional
+        chunk_size : int
             Size of the chunks to request from the server.  Default
-            256 kB.
+            256 kB
 
         Returns
         -------
         nodes : np.ndarray
-            Numpy array of nodes.
+            Numpy array of nodes
         """
         if self._chunk_size:
             chunk_size = self._chunk_size
@@ -307,14 +307,6 @@ class MeshGrpc(Mesh):
         request = anskernel.StreamRequest(chunk_size=chunk_size)
         chunks = self._mapdl._stub.Nodes(request)
         nodes = parse_chunks(chunks, np.double).reshape(-1, 3)
-
-        # Sometimes, MAPDL will return virtual, or hidden nodes.
-        # These appear to be at the beginning of the of the
-        # array. This behavior was identified in
-        # https://github.com/pyansys/pymapdl/issues/751
-        n_nodes = self.n_node
-        if n_nodes != nodes.shape[0]:
-            nodes = nodes[:n_nodes]
         return nodes
 
     @threaded

--- a/tests/test_mapdl.py
+++ b/tests/test_mapdl.py
@@ -140,55 +140,6 @@ def make_block(mapdl, cleared):
     mapdl.vmesh("ALL")
 
 
-@pytest.fixture(scope="function")
-def beam188_solve(mapdl, cleared):
-    # run non-interactive for speed
-    with mapdl.non_interactive:
-        mapdl.antype("STATIC")
-        mapdl.et(1, "BEAM188")
-        mapdl.keyopt(1, 3, 3)  # Cubic shape function
-        mapdl.keyopt(1, 9, 3)
-
-        # define material
-        mapdl.mp("EX", 1, 30E6)
-        mapdl.mp("PRXY", 1, 0.3)
-
-        # define section
-        w_f = 1.048394965
-        w_w = 0.6856481
-        sec_num = 1
-        mapdl.sectype(sec_num, "BEAM", "I", "ISection")
-        mapdl.secdata(15, 15, 28 + (2 * w_f), w_f, w_f, w_w)
-
-        # define geometry
-        for node_num in range(1, 6):
-            mapdl.n(node_num, (node_num - 1) * 120, 0, 0)
-
-        # define one node for the orientation of the beam cross-section
-        mapdl.n(6, 60, 1)
-
-        # define elements
-        for elem_num in range(1, 5):
-            mapdl.e(elem_num, elem_num + 1, 6)
-
-        # boundary conditions
-        mapdl.d(2, "UX", lab2="UY")
-        mapdl.d(4, "UY")
-        mapdl.nsel("S", "LOC", "Y", 0)
-        mapdl.d("ALL", "UZ")
-        mapdl.d("ALL", "ROTX")
-        mapdl.d("ALL", "ROTY")
-        mapdl.nsel("ALL")
-
-        # application of the surface load to the beam element
-        w = 10000 / 12
-        mapdl.sfbeam(1, 1, "PRES", w)
-        mapdl.sfbeam(4, 1, "PRES", w)
-
-        mapdl.run("/SOLU")
-        mapdl.solve()
-
-
 @pytest.mark.skip_grpc
 def test_internal_name_grpc(mapdl):
     assert str(mapdl._ip) in mapdl._name
@@ -1006,13 +957,3 @@ def test_cwd_directory(mapdl, tmpdir):
         mapdl.directory = wrong_path
         assert 'The working directory specified' in record.list[-1].message.args[0]
         assert 'is not a directory on' in record.list[-1].message.args[0]
-
-
-def test_handle_hidden_nodes(mapdl, beam188_solve):
-    # Sometimes, MAPDL will return virtual, or hidden nodes.
-    # These appear to be at the beginning of the of the
-    # array. This behavior was identified in
-    # https://github.com/pyansys/pymapdl/issues/751
-
-    # this should just run without error. Add in a trivial assert as best-practice
-    assert mapdl.mesh.grid.n_points == mapdl.mesh.n_node

